x-pack/plugins/observability/server/services/slo/find_slo_groups.ts

```diff
@@ -47,6 +65,18 @@ export class FindSLOGroups {
   constructor(private esClient: ElasticsearchClient, private spaceId: string) {}
   public async execute(params: FindSLOGroupsParams): Promise<FindSLOGroupsResponse> {
     const pagination = toPagination(params);
+    const groupBy = params.groupBy;
+    let groupByField;
+    switch (groupBy) {
+      case 'tags':
```

Comment:
If the value from the dropdown from the front end matched the name of the actual fields, you wouldn't need to do this switch statement or the other switch statement in the `group_list_view`. Consider switching the values in the form from `tags`, `status` and `sliType` to `slo.tags`, `status`, and `slo.indicator.type`.

Reply:
Good point, I rewrote this part